offline_docs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# Modify generated docs in a particular directory to work offline.
+
+date
+
+USAGE="Usage: offline_docs.sh [-d <dir-name>] [-r <dir-name>] [-z] [-o] [-h]."
+
+while getopts "d:r:zoh" opt; do
+  case $opt in
+    d)
+      HTML_DIR="$OPTARG"
+      ;;
+    r)
+      RESOURCE_DIR="$OPTARG"
+      ;;
+    z)
+      CREATE_ZIP="y"
+      ;;
+    o)
+      START_SERVER="y"
+      ;;
+    h)
+      echo "$USAGE"
+      exit 0
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG"
+      echo "$USAGE"
+      exit 1
+      ;;
+  esac
+done
+
+if [[ $HTML_DIR == "" ]] ; then
+  echo "No HTML directory specified. Defaulting to 'html'."
+  HTML_DIR="html"
+fi
+
+if [[ $RESOURCE_DIR == "" ]] ; then
+  echo "No resource directory specified. Defaulting to 'resources/offline'."
+  RESOURCE_DIR="resources/offline"
+fi
+
+printf "Warning: This will modify the contents of $HTML_DIR in place. Continue? y/N: "
```

Comment:
Might want to be more explicit here about what you mean by "in place." I suspect this means the script will modify whatever files are in the specified directory (or html, by default), but that's not totally clear. 
Maybe the script should be set to clone the docs repo (y/N) to get the latest version of the html files?